src/algebra/algebra/spectrum.lean

```diff
@@ -68,6 +68,14 @@ variable {R}
 noncomputable def resolvent (a : A) (r : R) : A :=
 ring.inverse (algebra_map R A r - a)
```

Comment:
I notice the notation `↑ₐ` is introduced for `algebra_map` below. Could we use it consistently through this file?

Reply:
done